lib/mongoid/tasks/database.rb

```diff
@@ -5,6 +5,18 @@ module Tasks
     module Database
       extend self
 
+      def create_collections(models = ::Mongoid.models)
+        models.each do |model|
+          next if model.storage_options.fetch(:collection_options, {}).empty?
+          if !model.embedded? || model.cyclic?
+            model.create_collection
+            logger.info("MONGOID: Created collection for #{model}:")
+          else
+            logger.info("MONGOID: collection options ignored on: #{model}, please define in the root model.")
```

Comment:
`create_indexes` method below does the same. I prefer consistency, it would be strange if two similar pieces of code have different structure. If we want to move the warning to `store_in`, the same should be done for `create_indexes`, and probably other methods here.

Reply:
Yep that is fine.